src/java/org/apache/cassandra/service/accord/repair/AccordRepair.java

```diff
@@ -150,20 +179,17 @@ private void repairRange(TokenRange range) throws Throwable
             }
             catch (RuntimeException e)
             {
-                // TODO Placeholder for dependency limit overflow
-//                dependencyOverflow = true;
-                cfs.metric.rangeMigrationDependencyLimitFailures.mark();
```

Comment:
why drop metric?

Reply:
I'd initially intended to keep accord migration and normal accord repairs separate, which didn't end up making sense. I'll rename and restore the metrics